core/controllers/base_test.py

```diff
@@ -671,7 +671,8 @@ def test_every_method_has_decorator(self):
             # Following handler are present in base.py where acl_decorators
             # cannot be imported.
             if (handler.__name__ == 'LogoutPage' or
-                    handler.__name__ == 'Error404Handler'):
+                    handler.__name__ == 'Error404Handler' or
+                    handler.__name__ == 'CsrfTokenHandler'):
```

Comment:
Replace this with `handler.__name__ in ('CsrfTokenHandler', 'Error404Handler', 'LogoutPage')`